docs/docs/tutorials.md

```diff
@@ -12,7 +12,9 @@
 -----------------------------------------------------------
 #### Adding Control Code
 - [Tutorials/ROS2 Messages and Services](Tutorials/ROS2/MessagesAndServices.md)
-- [Project Template](Tutorials/ROS2/Template.md)
+- [Controller Github Template (Python)](Tutorials/ROS2/PythonTemplate.md)
+- [Controller Github Template (C++)](Tutorials/ROS2/CPPTemplate.md)
+- [Linear Damper Example (Python)](Tutorials/ROS2/PythonLinearDamperExample.md)
```

Comment:
Do these two files exist yet? They're not added to this PR. Should commit at the same time as the files.